### DISCUSSION TITLE
How can I enable gesture scaling in the AR environment

### DISCUSSION THREAD

**Discussion by Johnmiicheal** [pmndrs/xr#354] [Q&A]
![image](https://github.com/user-attachments/assets/2591693e-5ba2-4bab-aed5-fcfdc09e21d3)
Basically, I want to be able to scale the model to whatever size i want inside the AR environment. Is there any way of doing that?
PS: Using a mobile phone

**Comment by bbohlender** [comment]
@Johnmiicheal Took some time, but we just released react-three/handle, which is basically build for use cases like this. Just wrap the car in a <Handle> and resizing and moving freely the car should work :)
